src/main/kotlin/no/nav/familie/ks/sak/api/FagsakController.kt

```diff
@@ -39,17 +42,42 @@ class FagsakController(private val fagsakService: FagsakService) {
     @PostMapping(produces = [MediaType.APPLICATION_JSON_VALUE])
     fun hentEllerOpprettFagsak(@RequestBody fagsakRequest: FagsakRequestDto): ResponseEntity<Ressurs<MinimalFagsakResponsDto>> {
         logger.info("${SikkerhetContext.hentSaksbehandlerNavn()} henter eller oppretter ny fagsak")
+
+        tilgangService.validerTilgangTilHandlingOgPersoner(
+            personIdenter = listOfNotNull(fagsakRequest.personIdent),
+            minimumBehandlerRolle = BehandlerRolle.SAKSBEHANDLER,
```

Comment:
Når brukes denne i frontend? gitt navnet ville jeg tro at minimumBehandlerRolle burde være veileder når man henter, men saksbehandler når man oppretter 🤔

Reply:
Reagerte på det selv, men det ble gjort sånn i ba-sak og, så lurer på om vi skal ha as-is oppførsel eller endre på det nå 🤔

Reply:
👍 Tenker siden det worst case er for strengt at vi kan la det være sånn